src/anomalib/engine/engine.py

```diff
@@ -139,8 +139,8 @@ def __init__(
         self.normalization = normalization
         self.threshold = threshold
         self.task = TaskType(task)
-        self.image_metric_names = image_metrics
-        self.pixel_metric_names = pixel_metrics
+        self.image_metric_names = image_metrics if image_metrics else ["AUROC", "F1Score"]
+        self.pixel_metric_names = pixel_metrics if pixel_metrics else ["AUROC", "F1Score"]
```

Comment:
Should we have default pixel metrics? What if someone trains a classification task?

Reply:
ah that's a good point!